src/ctrls/PlaybookBuildCtrl.js

```diff
@@ -29,6 +29,14 @@ class PlaybookBuildCtrl extends Controller{
             if(args[3] === 'usage') {
```

Comment:
- [ ] add task to backlog: use **yargs** for better args handling https://nodejs.org/en/knowledge/command-line/how-to-parse-command-line-arguments/